test/utils/config.go

```diff
@@ -251,3 +251,23 @@ Device:
 	}
 	return nil
 }
+
+func CheckChangesInLogs(t *testing.T, snap, expectedChange, expectedLog string, since time.Time) bool {
```

Comment:
Name: The function waits for something to appear in logs. It doesn't check for changes because it isn't comparing the current with past logs. `WaitForLogMessage`?
Parameters: `expectedChange` is unneeded. The function searches for `expectedLog` or rather the `expectedMessage`.

Reply:
How about `PollLogForMessage` as a more accurate function name?

Reply:
Polling has a different meaning: https://en.wikipedia.org/wiki/Polling_(computer_science)